shoop/simple_pricing/module.py

```diff
@@ -25,20 +25,13 @@ class SimplePricingModule(PricingModule):
     identifier = "simple_pricing"
     name = _("Simple Pricing")
 
-    pricing_context_class = SimplePricingContext
-
-    def get_context_from_request(self, request):
-        customer = getattr(request, "customer", None)
-
-        if not customer or customer.is_anonymous:
+    def get_customer_group_ids(self, customer):
```

Comment:
This should probably be a private (underscore-prefixed) API, or maybe preferably it should also live outside the class itself.